src/apps/organisations/__test__/components/OrgForm.test.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { Organisation } from "../../types";
+import { render, fireEvent, act } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { CONTEXT } from "../../../../utils";
+import OrganisationForm from "../../components/OrgForm";
+
+type orgFormProps = React.ComponentProps<typeof OrganisationForm>;
+
+let onSubmit = jest.fn();
+
+const testOrg: Organisation = {
+  id: "T1",
+  name: "Test organisation",
+  company: "Test company",
+  website: "https://test.com",
+  location: "Belgium",
+  public_access: "View",
+  extras: null
+};
+const baseProps: orgFormProps = {
+  onSubmit: onSubmit,
+  loading: true,
+  status: "",
+  errors: [],
+  context: CONTEXT.view,
+  savedValues: testOrg
```

Comment:
```suggestion
  savedValues: undefined
```
Since we want to testing creation, then we shouldn't pass testOrg in baseProps.